common/unsafe/src/test/scala/org/apache/spark/unsafe/types/CollationFactorySuite.scala

```diff
@@ -91,37 +91,35 @@ class CollationFactorySuite extends AnyFunSuite with Matchers { // scalastyle:ig
 
   test("fetch invalid UTF8_BINARY and ICU root locale collation names") {
```

Comment:
just for fun, I'm wondering what `getClosestSuggestionOnInvalidName("")` will return?